scripts/generate_release_markdown.sh

```diff
@@ -21,15 +21,11 @@ Release notes can be found here: https://docs.mattermost.com/install/desktop-app
 
 The download links can be found below.
 
-#### Windows - msi files (beta)
-$(print_link "${BASE_URL}/mattermost-desktop-${VERSION}-x64.msi")
-$(print_link "${BASE_URL}/mattermost-desktop-${VERSION}-x86.msi")
-
-#### Windows - setup exe files
-$(print_link "${BASE_URL}/mattermost-desktop-setup-${VERSION}-win.exe")
```

Comment:
We still technically build the EXE for automatic updates, but it's no longer being advertised to download. Users should always download the MSI for their system architecture.

@nab-77 Can I get your help coordinating with whoever runs the `mattermost.com` site to update this link to use the MSI for their architecture? It will require some code changes.